### ISSUE TITLE
A11y: Search autosuggestions do not display keyboard focus

### ISSUE THREAD

**Issue by ravenokko** [shopware/platform#8182]
### Shopware Version

6.7

### Affected area / extension

Platform(Default)

### Actual behaviour

It is possible to navigate the search suggestions with arrow keys but there is no focus indicator, which is a failure of [SC 2.4.7](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html)

### Expected behaviour

There should be a visual keyboard focus indicator as you navigate up/down the list of suggestions.

Note: This should work in all viewports.

### How to reproduce

Type a valid entry in the search that will prompt suggestions. Navigate with the up/down arrow keys.
